lib/linter/linter.js

```diff
@@ -1643,29 +1643,31 @@ class Linter {
         const options = normalizeVerifyOptions(providedOptions, config);
         const languageOptions = config.languageOptions;
 
-        languageOptions.ecmaVersion = normalizeEcmaVersionForLanguageOptions(
-            languageOptions.ecmaVersion
-        );
+        if (config.language === jslang) {
```

Comment:
How will languages other than JS perform the kind of option normalization that's done in this block?

Reply:
Currently, there's no way to do that. We could consider allowing languages to provide a `normalizeLanguageOptions()` method. Or maybe to return a value from `validateLanguageOptions()`, though the naming would be weird and seems better to just have `normalizeLanguageOptions()` that can throw an error if the config is invalid, otherwise return a normalized config.

Reply:
> Currently, there's no way to do that. We could consider allowing languages to provide a `normalizeLanguageOptions()` method. Or maybe to return a value from `validateLanguageOptions()`, though the naming would be weird and seems better to just have `normalizeLanguageOptions()` that can throw an error if the config is invalid, otherwise return a normalized config.

Both options seem reasonable. Shall we leave this if-block as it stands and open a new issue to discuss how to normalize language options?

Reply:
> Shall we leave this if-block as it stands and open a new issue to discuss how to normalize language options?

Sounds good to me.

Reply:
Alright, I'll file a new issue.

Reply:
This is a good catch. :+1: